sycl/include/sycl/backend.hpp

```diff
@@ -217,12 +217,17 @@ __SYCL_EXPORT queue make_queue(pi_native_handle NativeHandle,
                                const context &TargetContext,
                                const device *TargetDevice, bool KeepOwnership,
                                const async_handler &Handler, backend Backend);
-__SYCL_EXPORT queue make_queue_standard_or_immediate(
-    std::variant<ze_command_queue_handle_t, ze_command_list_handle_t>
-        NativeHandle,
-    const context &TargetContext, const device *TargetDevice,
-    bool KeepOwnership, const property_list &PropList,
-    const async_handler &Handler, backend Backend);
+
+// The make_queue2 and getNative2 functions are added as a temporary measure so
+// that the existing make_queue and getNative functions can co-exist with them.
+// At the next ABI redefinition the "2" versions will be removed and only
```

Comment:
NIT: please rephrase to make it clear that current "2" version will become the only "make_queue"

Reply:
OK, rephrased to be clearer that today's "make_queue2" and "getNative2" functions will be renamed to just "make_queue" and "getNative" at the next ABI change.